README.md

```diff
@@ -1,2 +1,26 @@
-# Hello-Microverse
-This is a hello microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-microverse
+
+> Hello microverse description.
+
```

Comment:
- [OPTIONAL] kindly add a clear description that explains your thousands of codes in a single paragraph. know that the README file is of great importance because it allows you to describe your application in detail.